messaging/src/test/resources/META-INF/maven/org.axonframework/axon-modelling/pom.properties

```diff
@@ -13,7 +13,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
```

Comment:
How come these changes snuck in?

Reply:
I deleted an re-added it. Will restore it to the original